### PR TITLE
Updating the keys in the web docs after glvis-4.0

### PR DESCRIPTION
**Pull request by tzanio**
Mostly replacing `i/I` in 2D with `o/O`

### CODE REVIEW

**Comment by tzanio** [comment]
~~This example: https://glvis.org/options-and-use/#glvis-scripts crashed on my Mac.~~

~~Do you know what the issue is?~~

Fixed by https://github.com/GLVis/glvis/pull/130

**Comment by tzanio** [comment]
Somewhat surprisingly for me I did not find any instances in MFEM, Laghos or Remhos that needed updating.

Let me know if I missed something...

**Comment by publixsubfan** [comment]
I pushed a fix for the script segfaulting issue at GLVis/GLVis#130.

**Comment by tzanio** [comment]
Thanks, that fixed it for me

**Review by v-dobrev — APPROVED** [review]

File: src/options-and-use.md [hunk 1]
```diff
@@ -296,7 +296,7 @@ quadrilateral into the unit square:
 
 The above plot was produced with:
 ```sh
-glvis -m quad.vtk -g quad-vec.gf -k "fRjlAmeIIiiiiiiiiiiibbvuuuuuuuuuuu"
+glvis -m quad.vtk -g quad-vec.gf -k "RjlAmeOOooooooooooobbvuuuuuuuuuuu************"
```
Comment (publixsubfan) [inline src/options-and-use.md:299]: Unrelated, but I noticed that 'u' and 'U' aren't documented in vsvector.cpp.

Comment (tzanio) [reply src/options-and-use.md:299]: Good catch, I'll add a fix in GLVis/glvis#130.

Comment (tzanio) [reply src/options-and-use.md:299]: See https://github.com/GLVis/glvis/pull/130/commits/dc01fe6d0c153f70cb232406a41a726365528e9d

**Review by publixsubfan — APPROVED** [review]

**Pull request merged**